dnf/cli/commands/check.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (C) 2016 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from dnf.i18n import _
+from dnf.cli import commands
+
+import argparse
+import dnf.exceptions
+
+
+class CheckCommand(commands.Command):
+    """A class containing methods needed by the cli to execute the check
+    command.
+    """
+
+    aliases = ('check',)
+    summary = _('check for problems in the packagedb')
+
+    @staticmethod
+    def set_argparser(parser):
+        parser.add_argument('--all', dest='check_types',
+                            action='append_const', const='all',
+                            help=_('show all problems; default'))
+        parser.add_argument('--dependencies', dest='check_types',
+                            action='append_const', const='dependencies',
+                            help=_('show dependencies problems'))
+        parser.add_argument('--duplicates', dest='check_types',
+                            action='append_const', const='duplicates',
+                            help=_('show duplicates problems'))
+        parser.add_argument('--obsoleted', dest='check_types',
```

Comment:
Just a minor cosmetic change. I would stick with yum's 'obsoletes' and also arguments would have united format, so user wouldn't have to remember that this argument ends with 'd' not 's'.